cmd/main.go

```diff
@@ -30,6 +35,7 @@ func main() {
 	}
 
 	if err = consulConfig.ServiceRegister(consulClient, agentConfig); err != nil {
+		log.Println("there")
```

Comment:
Do we need this?

Reply:
Well, not realy, thanks for your notice)